### PR TITLE
Add read-roi recipe

### PR DESCRIPTION
**Pull request by hadim**
Good to merge.

### CODE REVIEW

**Comment by hadim** [comment]
@jakirkham

**Comment by hadim** [comment]
Appveyor error is not related to the recipe (I think). I am re running it now.

File: recipes/read-roi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{% set name = "read-roi" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/hadim/read_roi
+  git_rev: {{ version }}
```
Comment (scopatz) [inline recipes/read-roi/meta.yaml]: Please do not use the git source, use a tarball instead

File: recipes/read-roi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+test:
+  imports:
+    - read_roi
+
+about:
+  home: https://github.com/hadim/read_roi/
+  license: BSD 3-Clause
+  summary: 'Read ROI files .zip or .roi generated with ImageJ.'
+  license_family: BSD
```
Comment (scopatz) [inline recipes/read-roi/meta.yaml]: Please also include the `license_file`

File: recipes/read-roi/meta.yaml (hunk 1 above)
Comment (hadim) [inline recipes/read-roi/meta.yaml]: Why that since there is the `git_rev` ? What is the difference since the integrity of the source should be the same.

**Comment by hadim** [comment]
Thank you for the review @scopatz

File: recipes/read-roi/meta.yaml (hunk 1 above)
Comment (scopatz) [inline recipes/read-roi/meta.yaml]: I am adding this to the docs, see https://github.com/conda-forge/conda-forge.github.io/pull/279

Comment (hadim) [inline recipes/read-roi/meta.yaml]: ok. good to me.

**Comment by hadim** [comment]
LGTM

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**